example/full/instance.properties

```diff
@@ -1350,3 +1350,16 @@ sleeper.default.statestore.committer.update.every.commit=false
 # When using the transaction log state store, this sets whether to update from the transaction log
 # before adding a batch of transactions in the asynchronous state store committer.
 sleeper.default.statestore.committer.update.every.batch=true
+
+# Default value for the concurrency assigned to the lambdas across the entire of the sleeper instance.
```

Comment:
I think this could be worded a bit more clearly. It could be interpreted as meaning that all the lambdas in the instance will share this concurrency. Maybe it would be better as "Default value for the reserved concurrency for each lambda within the Sleeper instance." ?

Reply:
Addressed